deps.ffmpeg/40-aom.zsh

```diff
@@ -148,7 +165,6 @@ fixup() {
         rm -rf ${target_config[output_dir]}/bin/libaom*.dll(N)
       }
       ;;
-    }
```

Comment:
Doesn't this match the opening brace for the `case` statement at line 118/135?

Reply:
Nope, that's what the next one does. Currently the script actually skips the stripping step below, removing this brace actually fixes it.

Reply:
I checked in a text editor, and now I understand. Thanks.